Tasks/PackerBuild/src/taskInputTemplateVariablesProvider.ts

```diff
@@ -46,7 +46,8 @@ export default class TaskInputTemplateVariablesProvider implements definitions.I
         this._templateVariables.set(constants.TemplateVariableLocationName, taskParameters.location);        
 
         var capturePrefix = tl.getVariable('release.releaseName') || tl.getVariable('build.buildnumber') || "vstscapture";
-        this._templateVariables.set(constants.TemplateVariableCapturePrefixName, capturePrefix);        
+        this._templateVariables.set(constants.TemplateVariableCapturePrefixName, capturePrefix);  
+        this._templateVariables.set(constants.TemplateVariableSkipCleanName, taskParameters.skipTempFileCleanupDuringVMDeprovision.toString());      
```

Comment:
yes, any extra variable which is not a parameter in template is neglected during apcker build process. I did not put it under a "isLinux" condition as it will only complicate code unnecessarily